linux_os/guide/system/software/integrity/fips/etc_system_fips_exists/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: fedora,rhcos4,ol7,ol8,rhel7,rhel8,rhv4,wrlinux1019
+prodtype: fedora,rhcos4,ol7,ol8,rhel7,rhel8,rhel9,rhv4,wrlinux1019
```

Comment:
I see you added no CCE to this rule, any reason for that?

Reply:
That rule is not in any profile, so it doesn't need a CCE. However, its check is used by a rule that is selected in a profile.